### PR TITLE
add ability to disable live queries

### PR DESCRIPTION
**Pull request by billcobbler**
Implements #2140 
- added toggle to disable live queries in advanced settings
- added new live query status endpoint (checks for disabled via config and Redis health)
- updated QueryPage UI to use new live query status endpoint

### CODE REVIEW

**Comment by anelshaer** [comment]
is there ability to enable it for specific users, or its only enable/disable for all users.

**Comment by billcobbler** [comment]
@anelshaer, it's just a simple global toggle and applies to all users but a sneaky admin user could toggle it off/on :smile:. 

Should be rather simple to add a bypass for admin users, but targeting specific users would be a larger change (I think) because I believe the only user segmentation that exists currently is admin/non-admin.

File: server/datastore/mysql/migrations/tables/20191220130734_AddLiveQueryDisabledToAppConfig.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+)
+
+func init() {
+	MigrationClient.AddMigration(Up20191220130734, Down20191220130734)
+}
+
+func Up20191220130734(tx *sql.Tx) error {
+	_, err := tx.Exec(
+		"ALTER TABLE `app_configs` " +
+			"ADD COLUMN `live_query_disabled` TINYINT(1) NOT NULL DEFAULT FALSE;",
```
Comment (groob) [inline server/datastore/mysql/migrations/tables/20191220130734_AddLiveQueryDisabledToAppConfig.go:14]: I suggest using a boolean here to improve readability.

File: server/kolide/app.go [hunk 2]
```diff
@@ -228,8 +231,9 @@ type OrgInfo struct {
 
 // ServerSettings contains general settings about the kolide App.
 type ServerSettings struct {
-	KolideServerURL *string `json:"kolide_server_url,omitempty"`
-	EnrollSecret    *string `json:"osquery_enroll_secret,omitempty"`
+	KolideServerURL    *string `json:"kolide_server_url,omitempty"`
+	EnrollSecret       *string `json:"osquery_enroll_secret,omitempty"`
+	LiveQueryDisabled *bool   `json:"live_query_disabled,omitempty"`
```
Comment (groob) [inline server/kolide/app.go]: gofmt?

File: server/datastore/mysql/migrations/tables/20191220130734_AddLiveQueryDisabledToAppConfig.go (hunk 1 above)
Comment (billcobbler) [reply server/datastore/mysql/migrations/tables/20191220130734_AddLiveQueryDisabledToAppConfig.go:14]: I was being consistent with the existing code base in that every other setting flag uses TINYINT(1) already.

File: server/kolide/app.go (hunk 2 above)
Comment (billcobbler) [reply server/kolide/app.go]: Fixed.

**Comment by zwass** [comment]
Follow up in #2181.

**Pull request merged**